shared/templates/template_OVAL_audit_rules_unsuccessful_file_modification_o_creat

```diff
@@ -54,7 +54,7 @@
       <value>^[\s]*-a[\s]+always,exit[\s]+(?:-F[\s]+arch=b64[\s]+)(?:-S[\s]+(?:[\S]+,)*({{{ SYSCALL }}})(?:,[\S]+)*)[\s]+</value>
   </constant_variable>
   <constant_variable id="var_audit_rule_{{{ SYSCALL }}}_o_creat_tail" version="1" datatype="string" comment="audit rule auid and key">
-    <value>[\s]+(?:-F\s+auid>={{{ auid }}}[\s]+)(?:-F\s+auid!=(unset|4294967295)[\s]+)(?:-k[\s]+|-F[\s]+key=)[\S]+[\s]*$</value>
+    <value>[\s]+(?:-F\s+auid>={{{ auid }}}[\s]+)(?:-F\s+auid!=(unset|4294967295)[\s]+)(?:-k[\s]+|-F[\s]+key=)unsuccesful-create[\s]*$</value>
```

Comment:
The hard coded key is aligned with the rules that use it.
But people may want to use different keys, see for example: https://github.com/ComplianceAsCode/content/pull/2715

AFAIK, the keys are there for usability reasons: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/security_guide/sec-defining_audit_rules_and_controls#bh-defining_file_rules_with_auditctl
> key_name is an optional string that helps you identify which rule or a set of rules generated a particular log entry.